torch/jit/frontend.py

```diff
@@ -178,6 +178,34 @@ def get_jit_class_def(cls, self_name):
     ctx = SourceContext(source, filename, file_lineno, leading_whitespace_len, False)
     return build_class_def(ctx, py_ast.body[0], methods, properties, self_name)
 
+def check_and_indent_multiline_strings(sourcelines):
```

Comment:
I think both the fn name and the description are missing a key point, that they indent only the triple-quoted text blocks found in sourcelines.  Also missing is a reason why this matters (might be good to mention that failure to properly indent comment blocks causes downstream dedent failure).

Reply:
oh, i forgot that multiline string is the technical term for the """ """ blocks.  So, i retract my comment.